extensions/amp-mowplayer/validator-amp-mowplayer.protoascii

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-mowplayer
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-mowplayer extension .js script"
+  requires: "amp-mowplayer"
```

Comment:
`satisfies`/`requires` pairs are no longer required for basic extension / tag pairs. There may be some out of date documentation somewhere, but please remove.

Reply:
Sure, Thanks!